src/custom-elements/src/components/date-picker/date-picker.tsx

```diff
@@ -155,6 +155,14 @@ export class DatePicker {
     this.value = date;
   }
 
+  /**
+   * Resets date
+   */
+  @Method()
+  async resetDate() {
+    this.value = null;
```

Comment:
@dani-cl-madrid Initially I used `undefined` but it throws a warning of a potentially infinite rerender.

Reply:
Null is ok.